criteria/common/pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>guava</artifactId>
       <version>${guava.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Is it possible to avoid runtime dependency on fudge ?

Reply:
Done